R/SoilProfileCollection-methods.R

```diff
@@ -80,12 +80,41 @@ setMethod(
     }
 
     # filter out missing data
-    h <- .data.frame.j(h, target.names, aqp_df_class(x))
+    h <- aqp:::.data.frame.j(h, target.names, aqp_df_class(x))
```

Comment:
You cannot use a vector of column names in the j index in the same way that you can in a data.frame. It might "work" but not the same way -- often it fails with a verbose error.